packages/fxa-payments-server/server/lib/amplitude.test.js

```diff
@@ -0,0 +1,26 @@
+const amplitude = require('./amplitude.js');
+const assert = require('chai').assert;
+
+const amplitudeConfig = {
+  disabled: false,
+};
+
+const testEvents = ['amplitude.sub.relyingPartyView'];
+// TODO: mock out logger to examine the fired log things.
+
+// TODO: test the snakeCase conversions: amplitude.sub.relyingPartyView -> amplitude.fxa_subscribe.relying_party_view
```

Comment:
Are you worried about these TODOs before merging?

Reply:
Yes, I'm working on the tests now